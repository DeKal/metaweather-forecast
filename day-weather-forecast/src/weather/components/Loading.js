import React from 'react'
import { CircularProgress, Grid } from '@material-ui/core'
import { makeStyles } from '@material-ui/core/styles'
import theme from 'core/styles/theme'

const useStyles = makeStyles(() => ({
  container: {
    textAlign: 'center'
  },
  loading: {
    marginTop: theme.spacing(4),
    color: theme.colors.disabledText
  }
}))

const Loading = () => {
  const classes = useStyles()
  return (
    <Grid className={classes.container}>
      <CircularProgress size={36} className={classes.loading} />
    </Grid>
  )
}

export default Loading
