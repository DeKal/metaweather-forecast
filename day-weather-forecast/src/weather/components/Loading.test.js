import React from 'react'
import { render } from '@testing-library/react'
import Loading from 'weather/components/Loading'

describe('Loading test', () => {
  test('should render correctly', () => {
    const { container } = render(<Loading />)
    expect(container).toMatchSnapshot()
  })
})
