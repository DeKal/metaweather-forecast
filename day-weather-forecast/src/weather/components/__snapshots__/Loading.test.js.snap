// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading test should render correctly 1`] = `
<div>
  <div
    class="MuiGrid-root makeStyles-container-1"
  >
    <div
      class="MuiCircularProgress-root makeStyles-loading-2 MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
      role="progressbar"
      style="width: 36px; height: 36px;"
    >
      <svg
        class="MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
          cx="44"
          cy="44"
          fill="none"
          r="20.2"
          stroke-width="3.6"
        />
      </svg>
    </div>
  </div>
</div>
`;
